packages/collector/test/tracing/protocols/graphql/test_definition.js

```diff
@@ -22,18 +22,18 @@ const globalAgent = require('../../../globalAgent');
 
 const agentControls = globalAgent.instance;
 
-function start(graphqlVersion) {
```

Comment:
Q: Why did you remove all of the logic? 🤔

Reply:
Restored the logic. I missed to consider the future 17 support.